recipes/add_dependencies.rb

```diff
@@ -16,14 +17,22 @@
 #    dependencies:
 #      - recipe:aws-parallelcluster::test_dummy
 #      - recipe:aws-parallelcluster-install::directories
-#      - resource:ec2_udev_rules
+#      - resource:package_repos:update
```

Comment:
Nitpick. I would keep both examples, the one with the default action and the one with the explicit action.

Reply:
Restored previous example too.